lib/ProcessFactory.php

```diff
@@ -29,9 +28,13 @@ public function run(string $command, ?Closure $callback = null): Process
         $process = $this->create($command);
         $process->run($callback);
 
-        if (! $process->isSuccessful()) {
-            throw new ProcessFailedException($process);
-        }
+        // This code is used to run the git checkout command.
+        // Even if it is not successful, there will be no critical damage in the end.
+        // Turning it off only makes debugging difficult.
+        // end currently blocking the build process
+        // if (! $process->isSuccessful()) {
+        //     throw new ProcessFailedException($process);
+        // }
```

Comment:
Ah, there's still one change I forgot to revert. I thought I did though. :thinking:

Reply:
Done.